packages/beacon-node/src/network/libp2p/index.ts

```diff
@@ -84,7 +84,9 @@ export async function createNodeJsLibp2p(
         },
       }),
     ],
-    streamMuxers: [mplex({maxInboundStreams: 256})],
+    // libp2p:mplex rate limit hit when receiving messages for streams that do not exist - closing remote connection
+    // libp2p:gossipsub:error Error: Too many messages for missing streams
+    streamMuxers: [mplex({maxInboundStreams: 256, disconnectThreshold: 50})],
```

Comment:
Had been getting a lot of following errors even after the removing the specific protocols. It seems once the identity process started, removing the protocols still cause this error. 

```
libp2p:gossipsub:error Error: Too many messages for missing streams
```